src/main/java/org/scijava/module/AbstractModule.java

```diff
@@ -76,11 +78,16 @@ public void cancel() {
 	public void initialize() throws MethodCallException {
 		// execute global module initializer
 		final Object delegateObject = getDelegateObject();
-		if (initializerRef == null) {
-			final String initializer = getInfo().getInitializer();
-			initializerRef = new MethodRef(delegateObject.getClass(), initializer);
+		if (delegateObject instanceof Initializable) {
+			((Initializable) delegateObject).initialize();
+		}
+		else {
```

Comment:
Do we need to do either/or here? If so, we should issue a warning when the module implements `Initializable` _and_ declares an initializer method. Or simpler: why not just run both initializers if both are available/specified?